ui/src/app/pages/profile/data-user-code-of-conduct.tsx

```diff
@@ -68,7 +69,8 @@ const styles = reactStyles({
 
 // TODO parameterize these
 const V4_PATH = '/data-user-code-of-conduct-v4.html';
-const V4_HEIGHT = '85rem';
+// via trial and error
+const V4_HEIGHT = '90rem';
```

Comment:
Good comment. Not for this PR, but I would like us to move away from the ALL_CAPS convention for constants.